pkg/crc/config/viper_config_test.go

```diff
@@ -121,7 +121,7 @@ func TestViperConfigLoadDefaultValue(t *testing.T) {
 
 	bin, err := os.ReadFile(configFile)
 	assert.NoError(t, err)
-	assert.JSONEq(t, `{"cpus":4}`, string(bin))
+	assert.JSONEq(t, `{}`, string(bin))
```

Comment:
maybe also add a comment here, `// default values are not written to config file`

Reply:
👍